sql/hive-thriftserver/src/main/java/org/apache/hive/service/ServiceUtils.java

```diff
@@ -18,7 +18,7 @@
 
 import java.io.IOException;
 
-import org.slf4j.Logger;
+import org.apache.spark.internal.Logger;
```

Comment:
The following code use it
I searched the `spark code base`. It seems that the `public static void cleanup(Logger log, java.io.Closeable... closeables)` method is not used. Maybe we can delete the following code?
https://github.com/apache/spark/blob/d7f69e7003a3c7e7ad22a39e6aaacd183d26d326/sql/hive-thriftserver/src/main/java/org/apache/hive/service/ServiceUtils.java#L55-L67